.github/workflows/check-issues-nightly.yml

```
@@ -0,0 +1,23 @@
+name: Check issues with pending validation labels
+
+on:
```

```suggestion
on:
  workflow_dispatch: {}
```

have you tried to execute this ci job? if not, maybe it's worth adding custom dispatch and trying it once.

---

Yeah I executed this locally by using [act](https://github.com/nektos/act)

---

It's still worth adding the `workflow_dispatch` option so we can run it with `et workflow-dispatch` command 😉

---

Sure, added